test/common/devCompileOptions.cmake

```diff
@@ -132,8 +132,8 @@ ELSE()
             LIST(APPEND alpaka_DEV_COMPILE_OPTIONS "-Wcast-align=strict")
         ENDIF()
 
-    # Clang or AppleClang
-    ELSEIF(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+    # Clang, AppleClang, ICPX
+    ELSEIF(CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR ${CMAKE_CXX_COMPILER_ID} STREQUAL "IntelLLVM")
```

Comment:
`IntelLLVM` is not known before CMake 3.20. It will probably still work with CMake 3.18 (the alpaka minimum) I guess.

Reply:
Since the code is the same for clang and icpx, it should not matter.